casa/OS/HostInfo.h

```diff
@@ -131,12 +131,22 @@ class HostInfo
     static ptrdiff_t swapFree();
     // </group>
 
+    // Introduce run-time memory limit re-specification capabilities
+    // <group>
+    static void setMemoryTotal(int memory);
```

Comment:
if you have changed them they are not reread from file
currently it is not possible to sanely restore the defaults as the aipsrc gets in the way of that, so this is almost mandatory